lbz/dev/server.py

```diff
@@ -142,7 +135,7 @@ def do_OPTIONS(self) -> None:  # pylint: disable=invalid-name
         self.handle_request()
 
 
-class MyDevServer:
+class MyDevServer(Thread):
```

Comment:
Can you describe in a few words what gives us this inheritance? I can investigate it on my own but I thought it will be simpler/faster to get it from you 😉